modules/migration/schemas/comment.json

```
@@ -0,0 +1,61 @@
+{
+  "title": "Comment",
+  "description": "Comments received on issues associated to a repository within a forge (Gitea, GitLab, etc.).",
+
+  "type": "array",
+  "items": {
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "issue_index": {
+        "description": "Unique identifier, relative to the issue.",
```

```suggestion
        "description": "Unique identifier, relative to the repository.",
```

Unless I'm mistaken the index is not relative to the issue but to the repository.

---

Thanks for the quick review!

I mistook the `issue_index` to be an identifier for the comment but upon further digging, I discovered that it identifies the issue. I will make the suggested change :+1: